dub.selections.json

```diff
@@ -2,26 +2,25 @@
 	"fileVersion": 1,
 	"versions": {
 		"agora": {"path":"submodules/agora/"},
-		"barcode": "0.6.1",
-		"base32": "0.1.0",
-		"bitblob": "1.1.4",
-		"botan": "1.12.19",
-		"botan-math": "1.0.3",
-		"d2sqlite3": "0.19.1",
-		"diet-ng": "1.7.4",
-		"dyaml": "0.8.3",
-		"eventcore": "0.9.13",
-		"libasync": "0.8.6",
-		"libsodiumd": "0.1.1+1.0.18",
-		"localrest": "~v0.x.x",
-		"memutils": "1.0.4",
-		"mir-linux-kernel": "1.0.1",
+		"barcode": {"path":"submodules/barcode/"},
+		"base32": {"path":"submodules/base32/"},
+		"bitblob": {"path":"submodules/bitblob/"},
+		"botan": {"path":"submodules/botan/"},
+		"d2sqlite3": {"path":"submodules/d2sqlite3/"},
+		"diet-ng": {"path":"submodules/diet-ng/"},
+		"dyaml": {"path":"submodules/dyaml/"},
+		"eventcore": {"path":"submodules/eventcore/"},
+		"libasync": {"path":"submodules/libasync/"},
+		"libsodiumd": {"path":"submodules/libsodiumd/"},
+		"localrest": {"path":"submodules/localrest/"},
+		"mir-core": "0.2.2",
```

Comment:
This needs to be removed before this PR can be ready. What pulls it in ?

Reply:
I haven't pushed the changes yet. Will do clean-up and rebase before I mark this as ready.